tests/index.test.ts

```diff
@@ -436,13 +436,10 @@ test('cjs exports', async () => {
 
 test('should error when file import cannot be found', async () => {
   await expect(() =>
-    rolldownBuild(
-      path.resolve(dirname, 'fixtures/unresolved-import/index.d.ts'),
-      [
-        dts({
-          emitDtsOnly: true,
-        }),
-      ],
-    ),
-  ).rejects.toThrow("Cannot resolve import './missing-file'")
+    rolldownBuild(path.resolve(dirname, 'fixtures/unresolved-import/ts.ts'), [
+      dts({
+        emitDtsOnly: true,
+      }),
+    ]),
+  ).rejects.toThrow("Could not resolve './missing-file'")
```

Comment:
The error message here is different because we're now deferring to Rolldown to emit the error, which is more descriptive.